tree/dataframe/src/RMetaData.cxx

```diff
@@ -9,84 +9,112 @@
  *************************************************************************/
 
 #include "ROOT/RDF/RMetaData.hxx"
+#include <nlohmann/json.hpp>
 #include <stdexcept> // std::logic_error
 
 namespace ROOT {
+
+namespace Internal {
+namespace RDF {
+
+struct RMetaDataJson {
+   nlohmann::json payload;
+};
+
+} // namespace RDF
+} // namespace Internal
```

Comment:
I would prefer a fully-qualified definition here
```suggestion
class Internal::RDF::RMetaDataJson {
   nlohmann::json payload;
};
```

(I see the rest of the file doesn't use that style, we could fix that in another PR)

Reply:
Thanks! I agree it's nicer here